db/schema/currency.ts

```diff
@@ -0,0 +1,11 @@
+import mongoose, { Schema } from 'mongoose';
+
+const currencySchema: Schema = new mongoose.Schema({
+  currency: {
```

Comment:
let's call this value. with this schema, for a currency record, to get the value i think it'll be accessed like this:
```
currencyRecord.getCurrency // something like this
```